bundles/org.openhab.binding.network/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -124,6 +124,12 @@
 				<description>Refresh time interval in minutes.</description>
 				<default>20</default>
 			</parameter>
+			<parameter name="initialDelay" type="integer" min="2">
+				<label>Initial Delay (min)</label>
```

Comment:
Unit in description is enough. If you feel this is necessary in addition, please use (Min) (all upper-case-letters as starting letters in labels)